src/test/resources/features/allergies.feature

```diff
@@ -0,0 +1,39 @@
+Feature: allergies feature
+
+  Background: 
+    Given User logs in, searches John doe and visits first patient dashboard
+    And User clicks on edit icon near allergies label
+
+  @run
+  Scenario: Launching allergies page
+    Then system loads patients allergies page
+
+  @run
+  Scenario: Add new allergies
+    And user clicks add new allergy button
+    Then system loads allergies page
```

Comment:
this should already be on the allergies page, you probably mean the add new allergy (allergy) page